TableViewKit/Protocols/Selectable.swift

```diff
@@ -18,7 +18,7 @@ extension Selectable {
     public func select(in manager: TableViewManager, animated: Bool, scrollPosition: UITableViewScrollPosition = .none) {
 
```

Comment:
I think that it's a good idea create a new method inside of manager to do this, in order to avoid access to tableView or delegate offside of manager.